optimism2/ovm2/get_contracts/get_contracts.sql

```diff
@@ -0,0 +1,20 @@
+CREATE SCHEMA IF NOT EXISTS ovm2;
+
+CREATE TABLE IF NOT EXISTS ovm2.get_contracts (
+	contract_address bytea UNIQUE,
+	contract_project text,
+	token_symbol text, 
+	contract_name text,
+	creator_address bytea, 
+	created_time timestamptz,
+	contract_creator_if_factory bytea,
+	is_self_destruct boolean
+);
+
+CREATE INDEX IF NOT EXISTS ovm2_get_contracts_contract_address_idx ON ovm2.get_contracts (contract_address);
+CREATE INDEX IF NOT EXISTS ovm2_get_contracts_created_time_idx ON ovm2.get_contracts (created_time);
+CREATE INDEX IF NOT EXISTS ovm2_get_contracts_contract_project_idx ON ovm2.get_contracts (contract_project);
+CREATE INDEX IF NOT EXISTS ovm2_get_contracts_contract_address_contract_project_idx ON ovm2.get_contracts (contract_address,contract_project);
+CREATE INDEX IF NOT EXISTS ovm2_get_contracts_contract_address_contract_project_created_time_idx ON ovm2.get_contracts (contract_address,contract_project,created_time);
```

Comment:
@MSilb7 index names have a max length of 63 characters in postgres. lines 18 & 19 are longer and get truncated, meaning they end up with the same name and error out. can you revisit naming standards and get all under 63 characters?

Reply:
updated!